src/vllm_tgis_adapter/tgis_utils/scripts.py

```diff
@@ -0,0 +1,237 @@
+from __future__ import annotations
+
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+from vllm.scripts import registrer_signal_handlers
```

Comment:
Do we actually need these signal handlers?

Reply:
Don't believe so, removed them